### PR TITLE
Callback wrapper

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "metasync",
-  "version": "0.1.6",
+  "version": "0.1.7",
```
Comment (aqrln) [inline package.json]: Unrelated change.

**Comment by tshemsedinov** [comment]
Landed in d18dab4
